### PR TITLE
Solution serialization redux

### PR DESCRIPTION
**Pull request by ernestask**
Since my last attempt backfired, I had to try a different approach.

### CODE REVIEW

**Comment by mkutlak** [comment]
```
afsjh
```

Noice.

**Comment by ernestask** [comment]
```
src/webfaf/utils.py:225:4: R0911: Too many return statements (13/12) (too-many-return-statements)
```

/me readies gun.

**Comment by ernestask** [comment]
FINALLY

Everything Now Should Be Working™

File: src/webfaf/utils.py [hunk 1]
```diff
@@ -227,57 +220,6 @@ def default(self, o): #pylint: disable=method-hidden
...
-        if isinstance(o, Problem):
-            d = {"id": o.id,
-                 "components": o.unique_component_names,
-                 "crash_function": o.crash_function,
-                 "bugs": [bug.url for bug in o.bugs],
-                 "status": o.status,
-                 "type": o.type,
-                 "reports": o.reports,
-                }
-            if hasattr(o, "count"):
```
Comment (ernestask) [inline src/webfaf/utils.py]: I dropped this, because I couldn’t figure out where it could ever come from.

**Pull request merged**